src/dfx/src/commands/deploy.rs

```diff
@@ -99,6 +101,23 @@ pub struct DeployOpts {
     /// Compute evidence and compare it against expected evidence
     #[arg(long, conflicts_with("by_proposal"))]
     compute_evidence: bool,
+
+    /// Transaction timestamp, in nanoseconds, for use in controlling transaction deduplication, default is system time.
+    /// https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/#transaction-deduplication-
+    //TODO(SDK-1331): unhide
+    #[arg(long, hide = true)]
+    created_at_time: Option<u64>,
```

Comment:
The `--created-at-time` parameter should require that the `canister_name` parameter is also present.  It could also be allowed if the project has only one canister, but I'm not sure it's worth the complexity.